keyboards/converter/HP_46010A/readme.md

```diff
@@ -0,0 +1,51 @@
+# HP 46010A
+
+![46010A](https://deskthority.net/w/images/a/a5/HP_46010A_--_top.jpg)
+
+A converter for the eponymous keyboard.
+
+Keyboard Maintainer: [Listofoptions](https://github.com/listofoptions)  
+Hardware Supported: HP 46010A, Teensy 2.0  
+
+Make example for this keyboard (after setting up your build environment):
+
+    make converter/HP_46010A:default
+
+included in this directory is a arduino program for teensy 2.0 (only one tested, might work on others) to grab a copy of the keyboard matrix.
+also included is a csv containing the matrix as an ordered by clock #, byte array
+    
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Could you replace this link block with this, instead: 

```markdown
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Reply:
ok so i assume you also want me to remove the arduino sketch too?

Reply:
Ideally, yes.  

If needed, dump it into a Gist and like that in the readme, for others to use.

Reply:
ok will do!